pkg/producer/protobuf/flow.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 VMware, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package github.com.vmware.goipfix.producer.protobuf;
+
+option go_package = "pkg/producer/protobuf";
+
+message FlowMessage {
+  // Header of IPFIX Message.
+  uint32 TimeReceived = 1;
```

Comment:
This file is for defining the Proto format, right? noticed that the number assigned is not in sequence order. how are the numbers defined?

Reply:
Yes these numbers signify the order of encoding of fields in Kafka message. Numbers did not follow a sequence because I added some fields later.